SharedProjects/Babylon2GLTF/GLTFExporter.Skin.cs

```diff
@@ -153,29 +156,31 @@ private GLTFSkin ExportSkin(BabylonSkeleton babylonSkeleton, GLTF gltf, GLTFNode
 
         private GLTFNode _exportBone(BabylonBone babylonBone, GLTF gltf, BabylonSkeleton babylonSkeleton, List<BabylonBone> bones)
         {
+            var boneNodePair = boneToGltfNodeMap.FirstOrDefault(pair => pair.Key.id.Equals(babylonBone.id));
+            if (boneNodePair.Key != null)
+            {
+                return boneNodePair.Value;
+            }
+
+            GLTFNode gltfNode = null;
             var nodeNodePair = nodeToGltfNodeMap.FirstOrDefault(pair => pair.Key.id.Equals(babylonBone.id));
             if (nodeNodePair.Key != null)
             {
                 return nodeNodePair.Value;
             }
 
-            var boneNodePair = boneToGltfNodeMap.FirstOrDefault(pair => pair.Key.id.Equals(babylonBone.id));
-            if (boneNodePair.Key != null)
-            {
-                return boneNodePair.Value;
-            }
 
-            // Node
-            var gltfNode = new GLTFNode
```

Comment:
if I'm not mistaken we never get here anymore, every node should be in the nodeToGltfNodeMap already

i think this whole function may be unnecessary

@drigax is there a case where a node or bone may be unrepresented by a BabylonNode, or does every node/bone have a matching BabylonNode?

Reply:
so I confirm that this method is not necessary
I comment out the method and his call.

Reply:
I don't think that this is true at all.

See this scene:
[SimpleSkinningCylinder.zip](https://github.com/BabylonJS/Exporters/files/3617813/SimpleSkinningCylinder.zip)

When exported, we have a babylon scene containing one node, the mesh that is skinned, while the bones are exported as part of the babylon skeleton. It appears that we consider skinning bones separately from nodes in this format, while in glTF skeletons are composed of nodes

Reply:
> glTF skeletons are composed of nodes

Yes, I can confirm that glTF skin joints point to node in the scene hierarchy.